adafruit_magtag/magtag.py

```diff
@@ -65,16 +70,16 @@ class MagTag(PortalBase):
     def __init__(
         self,
         *,
-        url=None,
-        headers=None,
-        json_path=None,
-        regexp_path=None,
-        default_bg=0xFFFFFF,
-        status_neopixel=None,
-        json_transform=None,
-        rotation=270,
-        debug=False,
-    ):
+        url: Optional[str] = None,
+        headers: Optional[Dict] = None,
```

Comment:
`Dict` is also a generic, and it also takes a `List` I believe so it should be `Optional[List[Dict[str, str]]]`